features/events/api/src/main/java/org/opennms/netmgt/xml/event/Correlation.java

```diff
@@ -108,6 +111,20 @@ public Correlation() {
         this._cueiList = new java.util.ArrayList<>();
     }
 
+    public static Correlation copyFrom(ICorrelation source) {
+        if (source == null) {
+            return null;
+        }
+
+        Correlation correlation = new Correlation();
+        correlation.setState(source.getState());
+        correlation.setPath(source.getPath());
+        correlation.getCueiCollection().addAll(MutableCollections.copyListFromNullable(source.getCueiCollection()));
```

Comment:
We don't need to use `copyListFromNullable()` here since we already have a new list in this object.

That method is meant to be used for fields that aren't already initialized so it creates a new list.

Double check if this is being done in the other JAXB objects.

Reply:
Good catch, done.